.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,65 @@
+name: Shellcheck Lint
+
+on:
+  push:
+    paths:
+      # Run workflow on every push
+      # only if a file within the specified paths has been changed:
+      - 'tests/**'
+      - 'usr/**'
+
+  pull_request:
+    paths:
+      # Run workflow on every push
+      # only if a file within the specified paths has been changed:
+      - 'tests/**'
+      - 'usr/**'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Shellcheck Lint
+
+    # This job runs on Linux
+    runs-on: ubuntu-latest
+
+    steps:
+      # Required to access files of this repository
+      - uses: actions/checkout@v2
+
+      # Download Shellcheck and add it to the workflow path
+      - name: Download Shellcheck
```

Comment:
Maybe instead of manually downloading shellcheck we can simply use a shellcheck docker image, e.g. the official one from https://hub.docker.com/r/koalaman/shellcheck? I think that would make it much simpler and less stuff for us to maintain here :-)

Reply:
I also would like to use the official `koalaman/shellcheck` docker container - but I didn't get it running on GitHub (on GitLab I only use the official `koalaman/shellcheck` docker container) 😢   

But there are some [GitHub Actions on the marketplace](https://github.com/marketplace?query=shellcheck+):
https://github.com/marketplace/actions/shellcheck-action or https://github.com/marketplace/actions/shellcheck will also show shellcheck warnings directly annotated in the GH GUI (if you like this - I personally prefer "oldschool"-shellcheck-output).
But they are also running on a "fat" Ubuntu-GitLab-Image and run than a docker container. This seems to be more overhead than downloading and running shellcheck directly in the "fat" Ubuntu-GitLab-Image provided by GH… 🤷🏻‍♂️